jwala-services/src/main/java/com/cerner/jwala/service/webserver/component/WebServerStateSetterWorker.java

```diff
@@ -102,15 +102,17 @@ public void pingWebServer(final WebServer webServer) {
             response = request.execute();
             LOGGER.debug("Web server {} status code = {}", webServer.getName(), response.getStatusCode());
 
-            if (response.getStatusCode() == HttpStatus.OK) {
+            if (HttpStatus.OK.equals(response.getStatusCode())) {
```

Comment:
HttpStatus.OK is enum OK(200, "OK"), will this be true or should we look for OK.value(), can you double check?